test/acceptance/tests/controller/controller_namespaces_test.go

```diff
@@ -0,0 +1,308 @@
+package controller
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/consul-helm/test/acceptance/framework"
+	"github.com/hashicorp/consul-helm/test/acceptance/helpers"
+	"github.com/hashicorp/consul/api"
+	"github.com/hashicorp/consul/sdk/testutil/retry"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	KubeNS                 = "ns1"
+	ConsulDestNS           = "from-k8s"
+	DefaultConsulNamespace = "default"
+
+	// The name of a service intention in consul is
+	// the name of the destination service and is not
+	// the same as the kube name of the resource.
+	IntentionName = "svc1"
+)
+
+// Test that the controller works with Consul Enterprise namespaces.
+// These tests currently only test non-secure and secure without auto-encrypt installations
+// because in the case of namespaces there isn't a significant distinction in code between auto-encrypt
+// and non-auto-encrypt secure installations, so testing just one is enough.
+func TestControllerNamespaces(t *testing.T) {
+	t.Skip()
```

Comment:
This will be deleted once 1.9.0 images are available

Reply:
By deleted I mean we will run these tests. Images in the tests will need to be updated as well!!

Reply:
👍